client/Index.jsx

```diff
@@ -14,12 +14,14 @@ class Index extends React.Component {
       amenities: [],
       hostAndRooms: {},
       thingsToDo: [],
+      listingId: window.location.href.split('/')[3] || 0,
     };
```

Comment:
try refactoring this to use window.location.pathname

Reply:
I tried but I didnt wanna deal with the trailing "/" , will do.